setup/setup.ml

```diff
@@ -765,11 +765,34 @@ value print_default_gwf_file conf =
       "long_date=no";
       "counter=no";
       "full_siblings=yes";
-      "hide_advanced_request=no" ]
+      "hide_advanced_request=no";
+      "perso_module_a=personnal_data";
+			"options_module_a=2";
+			"perso_module_b=parents";
+			"options_module_b=5";
+			"perso_module_c=unions";
+			"options_module_c=5";
+			"perso_module_d=siblings";
+			"options_module_d=3";
+			"perso_module_e=relations";
+			"options_module_e=2";
+			"perso_module_f=timeline";
+			"options_module_f=2";
+			"perso_module_g=notes";
+			"options_module_g=2";
+			"perso_module_h=sources";
+			"options_module_h=2";
+			"perso_module_i=trees";
+			"options_module_i=8";
+			"perso_module_j=gr_parents";
+			"options_module_j=2";
+			"perso_module_k=data_3col";
+			"perso_module_z=divider"
+    ]
   in
   let bname = try List.assoc "o" conf.env with [ Not_found -> "" ] in
   let dir = Sys.getcwd () in
-  let fname = Filename.concat dir (bname ^ ".gwf") in
+  let fname = Filename.concat dir (bname ^ ".gwb/params.gwf") in
```

Comment:
Yes, understood. Waiting for your solution for base architecture reorganisation.